drivers/wifi/nrf700x/osal/fw_if/umac_if/inc/fmac_cmd.h

```diff
@@ -13,6 +13,7 @@
 #define __FMAC_CMD_H__
 
 #define WIFI_NRF_FMAC_STATS_RECV_TIMEOUT 50 /* ms */
+#define WIFI_NRF_FMAC_PS_CONFIG_EVENT_RECV_TIMEOUT 50 /* ms */
```

Comment:
Try to shorten maro names where possible...in this case you could use CONF instead of CONFIG and EVNT instead of EVENT.